de1plus/plugins/visualizer_upload/plugin.tcl

```diff
@@ -17,16 +18,17 @@ proc upload {content} {
     set password $::plugins::visualizer_upload::settings(visualizer_password)
 
     set auth "Basic [binary encode base64 $username:$password]"
-    # For the moment we need to butcher the form-data format
-    set type "multipart/form-data; charset=utf-8;"
-
-    set headerl [list Authorization "$auth" Content-Type "$type"]
+    set boundary "--------[clock seconds]"
+    set type "multipart/form-data, boundary=$boundary"
```

Comment:
I think you should add `charset=utf-8` back

Reply:
okay, will do.

At some point we had the exact same structure, i changed it after Miha told me to remove the boundaries and just send data. Which worked for small files (forwhatever dark magic reason).